torch_ort_inference/torch_ort_inference/ortinferencemodule/provider_options.py

```diff
@@ -0,0 +1,29 @@
+class ProviderOptions:
```

Comment:
Please create a concrete class for `CPUProviderOptions` that inherits from `ProviderOptions`. And inherit `OpenVINOProviderOptions` from `CPUProviderOptions`.

If possible, make the provider argument in the base class as a required argument. Also, please use enums instead of strings where it makes sense.

Reply:
@baijumeswani, is there any reason that we want to have another concrete class for CPUProviderOptions? Is the intention here to have a separate concrete class for each device type?

Reply:
Oh, I was thought that was the intention. I thought that the CPUPorivderOptions would be the default provider options that ORTInferenceModule would use if not provided by the users. And users can override it by passing in an instance of the provider options that could be either openvino/cuda/rocm... provider options that we can introduce later. Was that not the intention behind creating ProviderOptions and OpenVinoProviderOptions?